src/dev/flang/tools/docs/Html.java

```diff
@@ -138,6 +138,34 @@ private String summary(AbstractFeature af)
   }
 
 
+  /**
+   * list of features that are redefined by feature af
+   * @param af
+   * @return list of redefined features, as HTML
+   */
+  private String redefines(AbstractFeature af)
+  {
+    if (af.redefines().isEmpty())
+      {
+        return "";
+      }
+
+    return "<div class='fd-redefines'><br />redefines: <br /><ul>" + redefines0(af) + "</ul><br /></div>";
+  }
+
+
+  private String redefines0(AbstractFeature af)
```

Comment:
Can you add a comment, in particular explaining  —if correct— that this is a helper for `redefines` and what this does.